### ISSUE TITLE
The results I change the data are not good

### ISSUE THREAD

**Issue by xiaqingsun** [adobe-research/custom-diffusion#34]
The results I reproduced with the data in the paper are good

my input images:
![image](https://user-images.githubusercontent.com/106377665/217413213-c76cb46b-7ad4-453a-b78c-cf852ad6d65e.png)

my result:
![image](https://user-images.githubusercontent.com/106377665/217413295-a5946039-82d9-4d0a-9510-a1e0e046105d.png)
![image](https://user-images.githubusercontent.com/106377665/217414374-7c3d8f1c-3077-4d1e-be5c-ee674adae4ad.png)

But I change other pictures, the result is worse（the sunglasses is not  like my input sunglasses）

my input images:
![image](https://user-images.githubusercontent.com/106377665/217413487-6f730ec7-9809-4584-8b30-084500fccca2.png)

my result:
![image](https://user-images.githubusercontent.com/106377665/217413547-9cef70ea-c98d-4f0e-935e-0b05813cf91c.png)

My training parameters are as follows：（I use the pretrained_model: runwayml/stable-diffusion-v1-5）

!accelerate launch src/diffuser_training.py \
--pretrained_model_name_or_path=$MODEL_NAME \
--output_dir=$OUTPUT_DIR \
--concepts_list="/content/concepts_list.json" \
--with_prior_preservation --prior_loss_weight=1.0 \
--resolution=512 \
--train_batch_size=2 \
--learning_rate=1e-5 \
--lr_warmup_steps=0 \
--max_train_steps=500 \
--num_class_images=200 \
--scale_lr --hflip \
--modifier_token "\<whxm\>+\<bkmj\>"

My concept_list.json are as follows:
![image](https://user-images.githubusercontent.com/106377665/217416034-46fe2260-8b33-458c-af60-8e57b27767ec.png)

Is there something wrong with my training process causing this poor result？
Thanks！

**Comment by nupurkmr9** [comment]
Hi, can you post if the individual sunglass generations with the fine-tuned model are similar to the target images? If that is not the case, maybe training longer with a lower learning rate can help.

**Comment by XavierXiao** [comment]
BTW, where to download the training data in the paper?

**Comment by xiaqingsun** [comment]
> 
![image](https://user-images.githubusercontent.com/106377665/217984193-00ef8bc4-ba2d-441e-ae1d-043a07ec4f34.png)
